.circleci/docker/build_docker.sh

```diff
@@ -46,7 +46,9 @@ fi
 # Build new image
 ./build.sh ${IMAGE_NAME} -t "${image}:${tag}"
 
-docker push "${image}:${tag}"
+if [ "${DOCKER_SKIP_PUSH:-false}" = "true" ]; then
```

Comment:
This should probably be reversed to `DOCKER_ALLOW_PUSH` but I wanted to keep the default behavior of the script just in case there's others that rely on the behavior of this script